src/pretalx/schedule/exporters.py

```diff
@@ -170,6 +179,13 @@ def get_data(self, **kwargs):
                     }
                     for room in self.event.rooms.all()
                 ],
+                "tracks": [
+                    {
+                        "name": str(track.name),
```

Comment:
Just a note: pretalx does not guarantee uniqueness of track names, the only fully reliable method of accessing tracks is by ID. (It's unlikely that events will have two tracks named the same, but … users.)

Reply:
yeah, thats fine for me.